perf/Makefile

```diff
@@ -5,33 +5,34 @@ clean:
 
 #-Wl,-rpath,$(TARGET_OSSL_LIBRARY_PATH)
 
-CFLAGS += -I$(TARGET_OSSL_INCLUDE_PATH) -I.
+CPPFLAGS += -I$(TARGET_OSSL_INCLUDE_PATH) -I.
+CFLAGS += -lpthread
```

Comment:
Strictly speaking, `-lpthread` should be part of `LDFLAGS`, not `CFLAGS`

Reply:
Thanks for cathing this. `-lpthread` is typo. I meant to write `-pthread`. I will re-test updated Makefile and push updated change.